doctor/serializers.py

```
@@ -0,0 +1,432 @@
+    
+    class Meta:
+        model = DoctorSchedule
+        fields = [
+            'id', 'doctor', 'doctor_name', 'weekday', 'weekday_display',
+            'start_time', 'end_time', 'visit_type', 'visit_type_display',
+            'max_patients', 'break_start', 'break_end',
+            'created_at', 'updated_at', 'is_active'
+        ]
+        read_only_fields = ['id', 'doctor_name', 'created_at', 'updated_at']
+    
+    def get_doctor_name(self, obj):
```

Ruff (0.12.2)

Missing return type annotation for public function `get_doctor_name`

(ANN201)